packages/field/src/css/field.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  colorsBackgroundLight,
+  colorsBorder,
+  colorsTextIcon,
+  layout,
+  type
+} from '@pluralsight/ps-design-system-core'
+
+import { sizes } from '../vars'
+
+export default {
+  '.psds-field__container': {
+    label: 'field__container',
+
+    display: 'inline-block'
+  },
+  '.psds-field__container--disabled': {
+    opacity: 0.5,
+    cursor: 'not-allowed'
+  },
+  '.psds-field__container--error': {},
+
+  //   '.psds-text-input__error': {
+  //     position: 'absolute',
+  //     right: `calc(-1 * (${iconWidths.medium} + ${layout.spacingXSmall}))`,
+  //     display: 'flex',
+  //     alignItems: 'center',
+  //     color: colorsStatus.error,
+  //     marginLeft: layout.spacingXSmall
+  //   }
```

Comment:
uh.....i forgot that i haven't added support for the error icon yet. will update